README.md

```diff
@@ -31,6 +31,9 @@ A list of companies that have paid Developer Community Writer Programs.
 - [Appsmith](https://blog.appsmith.com/launching-the-appsmith-writers-program) - $200-$400 per piece
   > Internal tools, low code, open-source, databases, application performance, engineering best practices, JavaScript, and Appsmith.
 
+- [Applozic](https://www.applozic.com/guest-writer/) - $100+ per piece
+  > Walkthrough of how to build Chat, Voice, and Video calling apps using Android, iOS or with frontend languages like Javascript frameworks and their respective ecosystems (Angular, Vue, React, 
```

Comment:
hello, alphabetically this is not in order. happy to merge when you fix this

Reply:
Hello @malgamves , I fixed the commit. Please review again.